packages/frontend/app/pods/components/pod/chat/chat-entry/component.ts

```diff
@@ -77,9 +78,14 @@ export default class ChatEntry extends Component<IArgs> {
   }
 
   @action onInput(event: KeyboardEvent) {
+    const EMOJI_REGEX = /:[^:]+:/g;
     const value = (event.target as any).value;
 
-    this.text = value;
+    if (value.match(EMOJI_REGEX)) {
```

Comment:
based on https://github.com/NullVoxPopuli/emberclear/pull/242

I think it's fast enough where we actually don't need to wrap this in a condition.

Also, I learned this today:
![image](https://user-images.githubusercontent.com/199018/66479673-f804ae00-ea6a-11e9-95f2-67091929c284.png)
So.. that's weird -- the regex caches the value and becomes a toggle

Reply:
ah, but you're using match